operator/config/rbac/role.yaml

```diff
@@ -4,6 +4,12 @@ kind: ClusterRole
 metadata:
   name: manager-role
 rules:
+- apiGroups:
```

Comment:
Do we need access to all resources?

Reply:
This is something that must be fixed. My goal is to start with this raw implementation that has follow-ups in it, but will enable us to work on them together in parallel.